tools/ci/actions/check-configure.sh

```diff
@@ -25,6 +25,15 @@
 
 set -e
 
+WANT="2.69"
```

Comment:
This can be adapted from https://github.com/ocaml/ocaml/blob/trunk/tools/pre-commit-githook#L105-L106:
```suggestion
WANT="$(sed -ne 's/^AC_PREREQ(\[\(.*\)\])/\1/p' configure.ac)"
```